### DISCUSSION TITLE
Any User Limitations? - Does Directory Connector allow more than 10 users?

### DISCUSSION THREAD

**Discussion by compumatter** [dani-garcia/vaultwarden#4078] [Q&A]
I have read when using with [Bitwarden and the Directory Connector](https://bitwarden.com/help/directory-sync-desktop/#:~:text=If%20you%20on%20the%20Teams,sync%20more%20than%2010%20members.) you are limited to 10 members using the Directory Connector and LDAP.  Can anyone confirm if such limitations apply to Vaultwarden.  

I am completely new to Vaultwarden.  Where [bitwarden has a "teams" and "enterprise"](https://bitwarden.com/pricing/business/) functionality perhaps these restrictions do not exist at all with Vaultwarden.  Just trying to get a handle on it.  Are there no user restrictions and no Directory Connector restrictions?

What sayeth the group?

**Comment by BlackDex** [comment]
The directory connector just uses the API, and as far as i know there are no restriction inside that tool.
There are also no restrictions for Bitwarden actually, you just need to pay for what you use.

Since Vaultwarden has no limits on the amount of users or organizations there should also be no limits for that tool.

**Reply by compumatter** [reply]
OK thank you for the response. I will install it and give it a go.
